source/Core/Threads/GUIThread.cpp

```diff
@@ -94,6 +94,44 @@ void gui_drawTipTemp(bool symbol, const FontStyle font) {
     }
   }
 }
+// Calibrate Cold Junction Compensation directly at boot, before internal components get warm.
+void performCJCC() {
+  while (preStartChecks() == 0) {
+    ulTaskNotifyTake(pdTRUE, TICKS_100MS);
+  }
+  uint32_t Temp = TipThermoModel::getTipInC();
+  if (!isTipDisconnected() || Temp <= 32) {
```

Comment:
```suggestion
  if (!isTipDisconnected() && Temp <= 32) {
```

Reply:
I tried that before, but in combination with the calibration checkbox, which unfortunately didn't work either.